client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -207,9 +203,9 @@ export const AppSchemaForm = ({ app }) => {
                   &nbsp;to request access.
                 </>
               )}
-            </Alert>
+            </Message>
           ) : (
-            <Alert color="info" isOpen={visible} toggle={onDismiss}>
+            <Message type="info" className="appDetail-error">
```

Comment:
Avoid quiet warning by using `SectionMessage`.

```suggestion
            <SectionMessage type="info" className="appDetail-error">
```